### ISSUE TITLE
Error: spawn netstat EMFILE on bittrex periodically

### ISSUE THREAD

**Issue by jyting** [BeerK0in/gunbot-monitor#28]
Hi,

When using gmon (defaults, no arguments) with bittrex, I periodically get this error.
```
events.js:182                                                                                                                                                                                                   
      throw er; // Unhandled 'error' event                                                                                                                                                                      
      ^                                                                                                                                                                                                         
                                                                                                                                                                                                                
Error: spawn netstat EMFILE                                                                                                                                                                                     
    at exports._errnoException (util.js:1016:11)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:189:19)
    at onErrorNT (internal/child_process.js:366:16)
    at _combinedTickCallback (internal/process/next_tick.js:102:11)
    at process._tickCallback (internal/process/next_tick.js:161:9)
```

What might be the cause? Happy to look into it and make a pull request with some guidance.

**Comment by BeerK0in** [comment]
What gmon version are you using?

**Comment by jyting** [comment]
I was using 0.32.0 at the time.

**Comment by BeerK0in** [comment]
That bug has been fixed in version 0.0.37
